packages/core/src/testability/testability.ts

```diff
@@ -90,6 +91,7 @@ export class Testability implements PublicTestability {
     private _ngZone: NgZone,
     private registry: TestabilityRegistry,
     @Inject(TESTABILITY_GETTER) testabilityGetter: GetTestability,
+    private readonly pendingTasks: PendingTasks,
```

Comment:
Should we refer to the public `ExperimentalPendingTasks` symbol here, since it makes it into the public API?
```suggestion
    private readonly pendingTasks: ExperimentalPendingTasks,
```

Reply:
I can't because I need the `hasPendingTasks` value which isn't exposed in the public symbol. I've updated this to use `inject` instead.